### PR TITLE
UHM-6440:  Remove Check-In and Vital Signs app from home page

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: configuration/pih/pih-config-haiti-cange.json [hunk 1]
```diff
@@ -26,9 +26,10 @@
     "socioEconomics",
     "systemAdministration",
     "todaysVisits",
+    "uhmVitals",
```
Comment (lnball) [inline configuration/pih/pih-config-haiti-cange.json:29]: What's the difference between the components:  uhmVitals and vitals?  Why does cange have uhmVitals but not other health facilities (belladere, boucan carre)?

File: configuration/pih/pih-config-hinche-ci.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "todaysVisits",
     "visitManagement",
     "visitNote",
+    "vitalsHomepageApp",
     "uhmVitals"
```
Comment (lnball) [inline configuration/pih/pih-config-hinche-ci.json:34]: Alphabetize ;)

File: configuration/pih/pih-config-haiti-thomonde.json [hunk 3]
```diff
@@ -31,6 +31,7 @@
     "visitManagement",
     "visitNote",
     "vitals",
```
Comment (lnball) [inline configuration/pih/pih-config-haiti-thomonde.json:33]: What's the difference between vitals and uhmVitals?

File: configuration/pih/pih-config-haiti-thomonde-ci.json [hunk 4]
```diff
@@ -41,6 +41,7 @@
     "visitManagement",
     "visitNote",
     "vitals",
```
Comment (lnball) [inline configuration/pih/pih-config-haiti-thomonde-ci.json:43]: same here.  vitals vs uhmVitals?

**Review by lnball — COMMENTED** [review]
Just ping me on Teams @mogoodrich .  Guessing you understand but it's not clear to me.

**Comment by mogoodrich** [comment]
Thanks @lnball  for creating the new ticket for fixing "uhmVitals" vs "vitals"... totally agree we should fix that... 

Also, I will fix the alphabetization issues you mentioned above in a follow-on commit.

**Pull request merged**